src/CPUSolver.cpp

```diff
@@ -782,23 +744,20 @@ void CPUSolver::addSourceToScalarFlux() {
   /* Loop over FSRs, energy groups */
   #pragma omp parallel for private(volume, sigma_t) schedule(guided)
   for (int r=0; r < _num_FSRs; r++) {
-
     volume = _FSR_volumes[r];
     sigma_t = _FSR_materials[r]->getSigmaT();
 
-      for (int e=0; e < _num_groups; e++) {
-        _scalar_flux(r,e) *= 0.5;
-        _scalar_flux(r,e) = FOUR_PI * _reduced_sources(r,e) +
-                            (_scalar_flux(r,e) / (sigma_t[e] * volume));
+    for (int e=0; e < _num_groups; e++) {
+      _scalar_flux(r,e) *= 0.5;
+      _scalar_flux(r,e) /= (sigma_t[e] * volume);
+      _scalar_flux(r,e) += (FOUR_PI * _reduced_sources(r,e));
     }
   }
-
-  return;
 }
 
 
 /**
- * @brief Computes the volume-weighted, energy integrated fission rate in
+ * @brief Computes the volume-averaged, energy-integrated fission rate in
```

Comment:
Yes, good call. I changed "fission rate" in the comments to "nu-fission rate" everywhere.